hazelcast/src/main/resources/hazelcast-config-3.10.xsd

```diff
@@ -2072,6 +2072,24 @@
                     </xs:documentation>
                 </xs:annotation>
             </xs:element>
+            <xs:element name="merge-policy" type="merge-policy" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation>
+                        While recovering from split-brain (network partitioning), CardinalityEstimator in the small cluster
+                        merge into the bigger cluster based on the policy set here.
+                        When an estimator merges into the cluster, an estimator with the same name might already exist in the cluster.
+                        The purpose of that estimator might be different for that same name. Which estimator should be kept?
+                        The conflict is resolved by the policy set here.
+                        <p>
+                            <br/>`com.hazelcast.spi.merge.PutIfAbsentMergePolicy` or `PUT_IF_ABSENT`:
```

Comment:
`PUT_IF_ABSENT` is wrong, I think this constant thing is only done supported by `ICache` or so. We should just put `PutIfAbsentMergePolicy` here (I left out the package name everywhere else, so the config looks as easy as possible to the user). The `DiscardMergePolicy` is missing. The path of the `HyperLogLogMergePolicy` is wrong.

I've aligned the documentation with the actual one from the generic `merge-policy` type:
```xml
                While recovering from split-brain (network partitioning), CardinalityEstimator in the small cluster
                merge into the bigger cluster based on the policy set here. When an estimator merges into the cluster,
                an estimator with the same name might already exist in the cluster.
                The merge policy resolves these conflicts with different out-of-the-box strategies.
                The out-of-the-box merge polices can be references by their simple class name.
                <p>
                    The out-of-the-box policies are:
                    <br/>DiscardMergePolicy: the estimator from the smaller cluster will be discarded.
                    <br/>HyperLogLogMergePolicy: the estimator will merge with the existing one, using the algorithmic merge for HyperLogLog.
                    <br/>PutIfAbsentMergePolicy: the estimator from the smaller cluster wins if it doesn't exist in the cluster.
                    <br/>The default policy is: HyperLogLogMergePolicy
                </p>
```

Reply:
thanks